neo/Prompt/Commands/Wallet.py

```diff
@@ -41,11 +47,19 @@ def execute(self, arguments):
         wallet = PromptData.Wallet
         item = get_arg(arguments)
 
-        # Create and Open must be handled specially.
+        # Create, Open, and Close must be handled specially.
         if item == 'create':
             self.execute_sub_command(item, arguments[1:])
             return
 
+        elif item == 'open':
+            self.execute_sub_command(item, arguments[1:])
+            return
+
+        elif item == 'close':
```

Comment:
I don't think so because this follows the original implementation (i.e. `wallet close` closes the open wallet)

Reply:
You added `close` as a "specially handled command" instead of a "normal command".

So for now, there is 
```
elif item == 'close':
    self.execute_sub_command(item)
    return
```
before
```
if not wallet:
    print("Please open a wallet")
    return
```

If you do not handle `close` a special way, it would be handled in 
```
try:
    self.execute_sub_command(item, arguments[1:])
except KeyError:
    print(f"Wallet: {item} is an invalid parameter")
```

Therefore we would have the 2 cases:
- a wallet is opened: works as now
- no wallet is opened: `print("Please open a wallet")` would be executed instead of nothing.

The complete implementation of `execute()` could be:
```
def execute(self, arguments):
    wallet = PromptData.Wallet
    item = get_arg(arguments)

    # Create, Open, and Close must be handled specially.
    if item in {'create', 'open'}:
        self.execute_sub_command(item, arguments[1:])
        return

    if not wallet:
        print("Please open a wallet")
        return

    if not item:
        print("Wallet %s " % json.dumps(wallet.ToJson(), indent=4))
        return

    try:
        self.execute_sub_command(item, arguments[1:])
    except KeyError:
        print(f"Wallet: {item} is an invalid parameter")
```

If it is still not clear (I don't find a good way to explain), let's talk on Discord later on.

Reply:
Okay, I think I understand. You are saying to just keep `close` as a normal subcommand. Okay. Lemme see what I can do.